openmc/universe.py

```diff
@@ -61,7 +63,9 @@ def __init__(self, universe_id=None, name=''):
         # Keys     - Cell IDs
         # Values - Offsets
         self._cell_offsets = OrderedDict()
-        self._num_regions = 0
+
+        if cells is not None:
```

Comment:
Yeah, I was thinking the same thing. If you don't mind, I'm going to hold off on this change as it may affect all the `get_all_cells()` methods and require a bit more refactoring.